iron-fit-behavior.html

```diff
@@ -123,8 +133,18 @@
       },
 
       /**
-       * The same as setting margin-top and margin-bottom css properties.
-       * @deprecated
```

Comment:
Yes, because now it is not the same as setting the margin, especially when setting 'verticalAlign = "auto"` where the user cannot know if to set `margin-top` or `margin-bottom`. In these cases the idea is to use `verticalOffset`